src/app/footer/footer.component.ts

```diff
@@ -15,19 +15,22 @@
  */
 import { HttpErrorResponse } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
 import { takeUntil } from 'rxjs/operators';
 
 import { MetadataService } from '../metadata/metadata.service';
 import { Base } from '../shared/base';
 import { Dockstore } from './../shared/dockstore.model';
 import { Metadata } from './../shared/swagger/model/metadata';
 import { FooterService } from './footer.service';
+import { Sponsor } from './sponsor.model';
 import { versions } from './versions';
 
 @Component({
   selector: 'app-footer',
   templateUrl: './footer.component.html',
   styleUrls: ['./footer.component.css'],
+  providers: [FooterService],
```

Comment:
What happens without this addtion?

Reply:
@garyluu nothing I think, will get rid of this